src/Microsoft.Framework.PackageManager/Restore/NuGet/HttpSource.cs

```diff
@@ -276,6 +285,19 @@ private static bool IsFileAlreadyOpen(string filePath)
             return false;
         }
 
+        private string GetCacheFilePath(string cacheKey)
```

Comment:
The reading logic is in `TryCache()` and it is calling this method, too: https://github.com/aspnet/dnx/blob/wei/invalidate-illegal-http-cache/src/Microsoft.Framework.PackageManager/Restore/NuGet/HttpSource.cs#L215

There is no duplicate of this logic.